packages/chip/src/Chip.types.ts

```diff
@@ -0,0 +1,40 @@
+import { AnchorHTMLAttributes } from 'react';
+
+export interface ComponentProps {
+  /**
+   * Customize Chip text color
+   */
+  variant?: 'filter' | 'choice';
+  /**
+   * The content of the Chip
+   */
+  children: React.ReactNode;
+  /**
+   * 	boolean to make chip disable.
+   */
+  disabled?: boolean;
+  /**
+   * selected style for choice chip
+   */
+  selected?: boolean;
```

Comment:
Hmmm. it looks like something strange.
how it should work ? 
we have two events on one component, click on delete icon and click on chip. do you have any particular cases for that ?

Reply:
both cases are handled in stories.

Reply:
resolved